frontend/packages/core/src/NPS/header.tsx

```diff
@@ -62,12 +62,16 @@ export const generateFeedbackTypes = (workflows: Workflow[]): SelectOption[] =>
       typeMap[group] = [];
     }
 
-    typeMap[group].push(
-      ...routes.map(route => ({
+    routes.forEach(route => {
+      typeMap[group].push({
         label: route.displayName || displayName,
         value: `/${path}/${route.path}`.replace(/\/\/+/g, "/"),
-      }))
-    );
+      });
+
+      if (get("route.componentProps.customNPS", [])) {
```

Comment:
Really minor but what do you think about something like `additionalNPS`? My only concern with custom is it's unclear they are being appended